src/Compilers/Core/CodeAnalysisTest/Collections/List/SegmentedList.Generic.Tests.Misc.cs

```diff
@@ -1059,7 +1059,7 @@ public static void ClearTests()
         }
 
         [Fact]
-        public static void TrueForAllTests()
+        public void TrueForAllTests()
```

Comment:
❓ Why not make all the tests in this file instance methods?

Reply:
📝 I tested locally and it appears to work fine to just mark these test classes `static`

Reply:
📝 I don't think we need to mark all test classes with static tests as `static`. However, for tests like this file which were copied from another repository (see the header comment for the file), marking the class `static` is a less intrusive change than marking all the methods as non-static.

Reply:
I missed the comment at the top. I went with this approach because it's more consistent with how we generally write tests. Most of our tests are not static.

Reply:
Reverted everything under src\Compilers.